src/accordion/accordion.tsx

```diff
@@ -39,20 +39,30 @@ const AccordionBase = ({
         );
     };
 
+    const renderTitleWrapper = () => {
+        if (!title && !enableExpandAll) {
+            return null;
+        }
+
+        return (
+            <TitleWrapper $hasTitle={!!title || showTitleInMobile}>
```

Comment:
`showTitleInMobile` is going to cause the same problem too, can we apply `display: none` if `showTitleInMobile=false` and `enableExpandAll=false`?

Reply:
- refactored css so that the flex properties do not need to rely on props
- added necessary props to apply `display: none` on mobile